python/ql/src/experimental/semmle/python/Frameworks.qll

```diff
@@ -7,5 +7,7 @@ private import experimental.semmle.python.frameworks.Django
 private import experimental.semmle.python.frameworks.Fabric
 private import experimental.semmle.python.frameworks.Flask
 private import experimental.semmle.python.frameworks.Invoke
+private import experimental.semmle.python.frameworks.MySQLdb
+private import experimental.semmle.python.frameworks.mysql
```

Comment:
There hasn't really been a 1-1 mapping between QL-library name and the provided python package name. I would like us sticking to the convention we already have with CamelCase like any other `qll` file (which is why it's `Flask` and not `flask`).

```suggestion
private import experimental.semmle.python.frameworks.Mysql
```

Reply:
Ah, happy to go with that.